### ISSUE TITLE
How to deploy the model to replicate.com

### ISSUE THREAD

**Issue by kiostudio** [MRzzm/DINet#84]
I would deploy the model to replicate.com for a more simple and convenient usage. 
Is there any guide or I am willing to pay for a sponsorship and donation to do so, thanks.

**Comment by Inferencer** [comment]
yeah ive done it for collab and hugging face etc just uploading and cleaning up files and braches etc
https://github.com/Inferencer/LipSick
